data/sql/updates/pending_db_world/rev_1692366711663255300.sql

```diff
@@ -0,0 +1,29 @@
+-- Table Condition - Fix Quests - 8353 - 8355 - 8356 - 8357 - 8311 --
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8353) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 12) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 12) AND (`ConditionValue2` = 0) AND (`ConditionValue3` = 0);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8355) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 12) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 12) AND (`ConditionValue2` = 0) AND (`ConditionValue3` = 0);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8356) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 12) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 12) AND (`ConditionValue2` = 0) AND (`ConditionValue3` = 0);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8357) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 12) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 12) AND (`ConditionValue2` = 0) AND (`ConditionValue3` = 0);
+
+DELETE FROM `conditions` WHERE (`SourceTypeOrReferenceId` = 19) AND (`SourceGroup` = 0) AND (`SourceEntry` = 8311) AND (`SourceId` = 0) AND (`ElseGroup` = 0) AND (`ConditionTypeOrReference` = 12) AND (`ConditionTarget` = 0) AND (`ConditionValue1` = 12) AND (`ConditionValue2` = 0) AND (`ConditionValue3` = 0);
+INSERT INTO `conditions` (`SourceTypeOrReferenceId`, `SourceGroup`, `SourceEntry`, `SourceId`, `ElseGroup`, `ConditionTypeOrReference`, `ConditionTarget`, `ConditionValue1`, `ConditionValue2`, `ConditionValue3`, `NegativeCondition`, `ErrorType`, `ErrorTextId`, `ScriptName`, `Comment`) VALUES
+(19, 0, 8311, 0, 0, 12, 0, 12, 0, 0, 0, 0, 0, 'Set the Quest available during the Hallows End event', NULL);
```

Comment:
This should be done in game_event_creature_quest

Reply:
I know @Kitzunu I had already seen it.

![image](https://github.com/azerothcore/azerothcore-wotlk/assets/1884642/2cb37e99-6fe3-4927-b254-0132d6c209ca)


The thing is: that all I did **(if you go and look in your local)** is change the ID of the Quest that was there..
Because before it was like this:
![image](https://github.com/azerothcore/azerothcore-wotlk/assets/1884642/048a4072-d982-499f-a43e-80180acfac5d)

And so were all 4 SUBs Quests.
I eliminated the SUBs and set the MAIN in their place that way.

And then, I set the 4 SUB dependent on MAIN

that's all..

Reply:
@Kitzunu 
Look, I did a check, and actually, both as I suspected and as you pointed out, that extra command was just an "iteration" of something that was already there, so I deleted it.